### DISCUSSION TITLE
OpenModal opening ModalPro

### DISCUSSION THREAD

**Discussion by ZARk-be** [Power-Components/livewire-powergrid#884] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

the ->openModal() method starts the free component. 
We are using the Pro version, and would love to have it started instead of adding the free component on top.

### Why is this needed?

Feel kinda left out having bought the Pro version :)
and adding both the pro and free version to a project isn't ideal

### Suggested Solution(s)

an ->openModalPro()  or an additional setting in the ->openModal() method ? It's basically just changing the event from OpenModal to modal.open

**Comment by soccerjf** [comment]
Not sure what is needed to "move" this request along, so if it's a matter of adding a comment, I agree with this request.  Kindly add the feature. Thank you.

**Comment by soccerjf** [comment]
Hack/Workaround
What I've done in the interim is set the row action button to "dispatch('edit',[_params_]) and then in the edit function added $this->dispatch('modal.open',component:'_pro modal file name_', arguments:[_same ones sent in the dispatch_] just started testing and haven't found any issues
